Dockerfile

```diff
@@ -83,25 +100,26 @@ RUN pip3 install --upgrade 'setuptools>=61' wheel && \
 
 COPY tools/docker/supervisord.conf /etc/supervisor/conf.d/nav.conf
 
+# Make an initial install of all NAV requirements into the virtualenv, to make
+# builds inside the container go faster
 COPY requirements/ /requirements
 COPY requirements.txt /
 COPY constraints.txt /
 COPY tests/requirements.txt /test-requirements.txt
 COPY doc/requirements.txt /doc-requirements.txt
-# Since we used pip3 to install pip globally, pip should now be for Python 3
-RUN pip-compile --resolver=backtracking --output-file /requirements.txt.lock -c /constraints.txt /requirements.txt /test-requirements.txt /doc-requirements.txt
-RUN pip install -r /requirements.txt.lock
+RUN --mount=type=cache,target=/source/.cache \
+    cd /opt/venvs/nav && \
+    pip-compile --resolver=backtracking --output-file ./requirements.txt.lock -c /constraints.txt /requirements.txt /test-requirements.txt /doc-requirements.txt ; \
+    pip install -r ./requirements.txt.lock
 
 ARG CUSTOM_PIP=ipython
-RUN pip install ${CUSTOM_PIP}
+RUN --mount=type=cache,target=/source/.cache \
+    pip install ${CUSTOM_PIP}
 
 COPY tools/docker/full-nav-restore.sh /usr/local/sbin/full-nav-restore.sh
 
-# Set up for mounting live source code from git repo at /source
-RUN    git config --global --add safe.directory /source
 VOLUME ["/source"]
 ENV    DJANGO_SETTINGS_MODULE nav.django.settings
-EXPOSE 80
+EXPOSE 8080
```

Comment:
What effect will this change have? Will nav be running on 80 or 8080?

Reply:
As Ilona said, this is all explained in the actual commit log of the change.